internal/theme/theme.go

```diff
@@ -0,0 +1,13 @@
+package theme
```

Comment:
Change the package name from theme to colormode, as theme sounds too general. Also rename the directory name.